firmware/Makefile

```diff
@@ -14,8 +14,7 @@ clean: FORCE
 deepclean: clean
 ifeq ($(findstring stm32,$(PLATFORM)),stm32)
 	@echo "Deleting files and directories in $(CUBEMX_DIR) that are ignored by Git"
-	@cd $(CUBEMX_DIR)
-	@find | xargs git check-ignore | xargs rm -rf
+	find $(CUBEMX_DIR) | xargs git check-ignore | xargs rm -rf
```

Comment:
Would suppressing the find command from printing to the terminal be better to keep it clutter free or no?

Reply:
The command doesn't have any console output, so I think it's nice to show what is being executed

Reply:
Agree with Blake, the added verbosity is nice to have since the command doesn't output anything to the console